### PR TITLE
628 modify free kick for penalty kick

### PR DESCRIPTION
**Pull request by Vuwij**
- [ ] Only the minimal change is included in this PR the commits are kept minimal
- [ ] Code is readable and documented in the code
- [ ] Minimal tests have been added

### CODE REVIEW

File: .idea/soccerbot.iml [hunk 1]
```diff
@@ -14,6 +14,7 @@
       <excludeFolder url="file://$MODULE_DIR$/docs" />
       <excludeFolder url="file://$MODULE_DIR$/.github" />
       <excludeFolder url="file://$MODULE_DIR$/.pytest_cache" />
+      <excludeFolder url="file://$MODULE_DIR$/venv" />
```
Comment (Vuwij) [inline .idea/soccerbot.iml:17]: remove this, dont use venv

File: soccer_strategy/src/soccer_strategy/robot_controlled.py [hunk 2]
```diff
@@ -29,6 +32,7 @@ def __init__(
         self.max_kick_speed = 2
         self.navigation_goal_localized_time = rospy.Time.now()
         self.kick_with_right_foot = True
+        self.kick_angle_pub = rospy.Publisher("strategy/kick_ang", Range, queue_size=1, latch=True)
```
Comment (Vuwij) [inline soccer_strategy/src/soccer_strategy/robot_controlled.py:35]: just call the topic kicking_range

File: soccer_strategy/src/soccer_strategy/robot_controlled.py [hunk 3]
```diff
@@ -93,6 +97,12 @@ def can_kick(self, ball: Ball, goal_position):
         nav_angle_diff = player_angle - robot_ball_angle
         distance_of_player_to_ball = np.linalg.norm(player_position - ball_position)
 
+        header = Header()
```
Comment (Vuwij) [inline soccer_strategy/src/soccer_strategy/robot_controlled.py:100]: add comment on top of this code block

File: soccer_strategy/src/soccer_strategy/robot_controlled.py [hunk 4]
```diff
@@ -93,6 +97,12 @@ def can_kick(self, ball: Ball, goal_position):
...
+        header.stamp = rospy.Time.now()
+        header.frame_id = f"robot{self.robot_id}/torso"
+        ang = Range(header, 1, abs(nav_angle_diff), 1, 2, 2)
```
Comment (Vuwij) [inline soccer_strategy/src/soccer_strategy/robot_controlled.py:103]: can't really understand the variables so maybe do 
Range(header=header, blabla=1, angle=abs(nav_angle_diff)) etc for better readibility

File: soccer_strategy/src/soccer_strategy/robot_controlled.py (hunk 2 above)
Comment (Vuwij) [reply soccer_strategy/src/soccer_strategy/robot_controlled.py:35]: and call the publisher self.kicking_range_publisher

File: test_integration.py [hunk 5]
```diff
@@ -17,6 +17,9 @@
 from soccer_msgs.msg import BoundingBoxes, RobotState
 from soccer_strategy.src.soccer_strategy.team import Team
 
+from sensor_msgs.msg import Range
```
Comment (Vuwij) [inline test_integration.py:20]: dont think these are needed

File: test_integration.py [hunk 6]
```diff
@@ -178,9 +181,10 @@ def test_kick_right(self):
             gt_ball_pose = self.get_ball_pose(gt=False)
             if gt_ball_pose is not None:
                 printlog(f"Current ball location: {gt_ball_pose}")
+
                 if gt_ball_pose[0] > 4.5:
                     printlog("Goal Scored")
-                    return
+                    #return
```
Comment (Vuwij) [inline test_integration.py:187]: what is this for?

File: soccer_strategy/src/soccer_strategy/robot_controlled.py [hunk 7]
```diff
@@ -3,11 +3,13 @@
 
 import numpy as np
 import rospy
+import rviz
```
Comment (Vuwij) [inline soccer_strategy/src/soccer_strategy/robot_controlled.py:6]: not needed

File: soccer_strategy/src/soccer_strategy/robot_controlled.py (hunk 4 above)
Comment (Vuwij) [reply soccer_strategy/src/soccer_strategy/robot_controlled.py:103]: or I prefer

r = Range()
r.header.stamp = rospy.Time.now()
r.header.frame_id = f"robot{self.robot_id}/torso"
r.blabla = 1

for better readability

**Comment by sumiyuan** [comment]
Removed unused imports, and fixed conventions.

**Review by sumiyuan — APPROVED** [review]

**Pull request merged**